src/pages/Lend/Filter/LendFilterPage.vue

```diff
@@ -143,7 +143,7 @@ function isFLSSEligible(route = {}) {
 	// check route for eligibility
 	const eligibleQueryParams = ['page', 'sortBy', 'gender', 'registration'];
 	const queryParamKeys = Object.keys(route?.query);
-	const allowedSorts = ['expiringSoon', 'loanAmountDesc', 'loanAmount', 'popularity'];
+	const allowedSorts = ['expiringSoon', 'loanAmountDesc', 'loanAmount']; // 'popularity'
```

Comment:
Do we also need to remove "Recommended" from the available alpha sorts in the UI?

Reply:
Recommended is the default FLSS sort. I never actually went through mapping popularity to recommended which was good b/c we determined that wouldn't be what we'll do. Popularity is distinct. We do have a switch to flip to "popularity" when the query type isn't FLSS but I don't see a way that would flip at this time.

Reply:
Also, "Recommended" is the Label mapped from "personalized" for FLSS.

Reply:
Oh right, recommended maps to personalized, not popularity.